index.html

```diff
@@ -124,41 +148,49 @@ <h3 class="work-header">Multi-Post Stories</h3>
             <button type="button" class="work-card-button">See Project</button>
-          </article>        
-        </section>
-        <section class="contact-us">
-          <div class="contact-us-text">
-            <h2 class="contact-header">CONTACT US</h2>
-            <p class="contact-p">If you have an application you are interested in developing, a feature that you need built or a project that
-              needs coding. I'd love to help with it!</p>
           </div>
-          <form action="https://formspree.io/f/mayvvvrn" method="post" class="contact-form">
-            <ul class="contact-inputs">
-              <li>
-                <input class="small-text" type="text" name="user_name" placeholder="First and Last Name" maxlength="30" required>
-              </li>
-              <li>
-                <input class="small-text" type="email" name="user_email" value="example1@gmail.com" required>
-              </li>
-              <li>
-                <textarea class="msg" name="user_message" maxlength="500" placeholder="Enter the message here" required></textarea>
-              </li>
-              <li class="submit-container">
-                <button class="contact-submit" type="submit">Send</button>
-              </li>
-            </ul>
-          </form>
-        </section>
-        <footer class="footer">
-          <div class="social-media">
-            <button type="button" class="social-icon"><img src="multimedia/githubicon.png" alt="Github cat icon"></button>
-            <button type="button" class="social-icon"><img src="multimedia/linkedinicon.png" alt="Linkedin icon"></button>
-            <button type="button" class="social-icon"><img src="multimedia/angelist1.png" alt="Peace hand sign icon"></button>
-            <button type="button" class="social-icon"><img src="multimedia/twittericon.png"
-                alt="Twitter birdie icon"></button>
-            <button type="button" class="social-icon"><img src="multimedia/Icon-Medium.png" alt="Medium icon"></button>
-          </div>
-          <p>karlavdelgadof@gmail.com</p>
-        </footer> 
+        </article>        
+      </section>
+      <section class="contact-us">
+        <div class="contact-us-text">
+          <h2 class="contact-header-hidden">Let's chat!</h2>
+          <h2 class="contact-header">CONTACT US</h2>
+          <p class="contact-p">If you have an application you are interested in developing, a feature that you need built or a project that
+              needs coding. I'd love to help with it!</p>
+        </div>
+        <form action="https://formspree.io/f/mayvvvrn" method="post" class="contact-form">
+          <ul class="contact-inputs">
+            <li>
+              <input class="small-text" type="text" name="user_name" placeholder="First and Last Name" maxlength="30" required>
+            </li>
+            <li>
+              <input class="small-text" type="email" name="user_email" value="example1@gmail.com" required>
```

Comment:
I apologize if no reviewer previously mentioned this, but you should remove the `value` attribute and change it to a `placeholder` attribute with a text similar to what you have for the name input. Kindly fix this as having a hard-coded `value` provides a strange UX.